bin/webpack-entries.js

```diff
@@ -178,4 +178,8 @@ const getEntryConfig = ( type = 'main', exclude = [] ) => {
 
 module.exports = {
 	getEntryConfig,
+
+	externals: {
+		'@wordpress/edit-site': 'wp.editSite',
+	},
```

Comment:
`@wordpress/edit-site` is [already setup as an external](https://github.com/WordPress/gutenberg/blob/fa4f5a5e4e5a1c1e1309f59ff9a99526f92a0a6a/packages/dependency-extraction-webpack-plugin/lib/util.js#L45-L50) via `@wordpress/dependency-extraction-webpack-plugin`. This is unnecessary.